2022/incoming/lapland-packaging-department-pt2.pod

```diff
@@ -0,0 +1,220 @@
+Author: Csaba Simándi <x1m4nd1@gmail.com>
+Title: Lapland Packaging Department - Part 2
+Topic: Carton aka Bundler for Perl
```

Comment:
```suggestion
Topic: Carton
```

Ideally the topic should be just the name of the module being covered. That doesn't work for your first article, but I think it works here.